runtime/data/test_data.csv

```diff
@@ -6,4 +6,6 @@
 11,2013-09-10,09:19:50,13:55:54
 11,2013-09-11,09:13:26,16:15:27
 11,2013-09-12,10:18:36,16:41:25
-11,2013-09-13,13:16:56,15:04:02
\ No newline at end of file
+11,2013-09-13,13:16:56,15:04:02
+11,2013-09-13,13:16:56,150402

```

Comment:
On a second thought, let's keep this file consistent. Leave it untouched and add broken records on demand in a particular test.